doc/lua_api.txt

```diff
@@ -5240,6 +5240,8 @@ This is basically a reference to a C++ `ServerActiveObject`
     * in first person view
     * in third person view (max. values `{x=-10/10,y=-10,15,z=-5/5}`)
 * `get_eye_offset()`: returns `offset_first` and `offset_third`
+* `send_mapblock(blockpos)`: returns `false` if failed. Sends a forced
+    mapblock to the remote player.
```

Comment:
Not necessary a forced mapblock. `Sends a server-side loaded mapblock to the player.`

Reply:
Updated. Check the latest version.